exporters/zipkin/src/main/java/io/opentelemetry/exporter/zipkin/ZipkinSpanExporterBuilder.java

```diff
@@ -19,10 +24,15 @@
 
 /** Builder class for {@link ZipkinSpanExporter}. */
 public final class ZipkinSpanExporterBuilder {
+
+  private static final Logger logger = Logger.getLogger(ZipkinSpanExporterBuilder.class.getName());
+
   private BytesEncoder<Span> encoder = SpanBytesEncoder.JSON_V2;
   @Nullable private Sender sender;
   private String endpoint = ZipkinSpanExporter.DEFAULT_ENDPOINT;
   private long readTimeoutMillis = TimeUnit.SECONDS.toMillis(10);
+  @Nullable private InetAddress localIpAddress;
+  private boolean localIpAddressSet = false;
```

Comment:
Alternatively you could initialize `localIpAddress` to the result of `produceLocalIp()` and omit this.